lib/internal/process.js

```diff
@@ -1,14 +1,7 @@
 'use strict';
 
 const errors = require('internal/errors');
-var _lazyConstants = null;
-
-function lazyConstants() {
-  if (!_lazyConstants) {
-    _lazyConstants = process.binding('constants').os.signals;
-  }
-  return _lazyConstants;
-}
+var lazyConstants;
```

Comment:
Re https://github.com/nodejs/node/pull/14167#issuecomment-314807428 comment this could be inlined.